docs/txs/Feepayer.md

```diff
@@ -0,0 +1,54 @@
+#Fee payer
+
+When a tx is submitted someone has to pay the fee to get it into the chain.
+
+The order of who pays goes:
+1) `tx.auth_info.fee.granter` -> address of a valid grant that will pay the fee
+2) `tx.auth_info.fee.payer` -> address of a signer who IS NOT the first signer that will pay the fee
+3) `tx.auth_info.signer_infos[0]` -> address of the first signer in the list
+   1) Can be first signature if signer_info is not present -> the indices remain the same
+
+Note: There will also be additional msg-based fees to handle
+
+##Fees
+
+Base tx fees come through as nhash, but additional fees/tips can come through as different coin.
+This should be handled appropriately.
+
+## Code Changes
+
+### DB Changes
+* Add `tx_feepayer`
+  * height
+  * tx info
+  * payer_type -> granter, payer, first_signer
+  * address
+
+* Add `tx_fee`
+  * NOTE: This will move the fees away from coming from the tx data
+  * height
+  * tx info
+  * fee_type -> base, msg based, base overage, priority
+  * denom
+  * amount
+
+* Added scripts in migration to add existing fees and feepayers to the tables.
+* Will need to manually run updates on txs without a 'signer_info' address
+
+### Ingestion
+* Add grpc for `msg_based_fees` -> will give param for min_gas_price (as currently hardcoded)
```

Comment:
nit: believe we call it floor_gas_price, since there is a param on each node for called min_gas_price set by validator/node. thought i would clarify.

Reply:
good to know.